repository/MonticelloFileTree-Core.package/MCFileTreeRepository.class/instance/readerClassForFileNamed..st

```diff
@@ -0,0 +1,4 @@
+actions
+readerClassForFileNamed: fileName
+	^ "MCReader readerClassForFileNamed: fileName"
+		MCFileTreeReader
```

Comment:
Yeah, for the moment I don't want to bother with filename heuristics trying to guess which version/variant gets used, so I left `MCFileTreeRepository` and the old hierarchy untouched, and the new code fixes the expected format explicitly by returning `MCFileTreeReader`. To test, I create a `cypress://` repo (instead of `filetree://`) in Monticello Browser, and I'm done.